README.md

```diff
@@ -39,8 +39,8 @@ Once you've set your environment information below you'll find the commands you'
 1. Create and activate a python environment:
```

Comment:
on lines 13 to 23 there were some instructions in need of an update due to changing the `.example.config` to `.env.example` file. other than that the PR is perfect @bajcmartinez

Reply:
I totally missed that, thanks for noticing, it is good to go now!

Reply:
I think we are good now 😉